app/javascript/src/Policies/types/Policies.jsx

```diff
@@ -7,7 +7,7 @@ export type RawPolicy = {
   version: string,
   description?: string,
   summary?: string,
-  configuration: Object
+  configuration: {}
```

Comment:
If not possible right now to create a type for configuration, better to leave the `Object` type.
And as configuration or "data" below goes, you could create some approximation of a type or mark as a TODO